core/storage/topic/gae_models.py

```diff
@@ -166,6 +166,11 @@ class TopicModel(base_models.VersionedModel):
     # Multiply and Divide'.
     page_title_fragment_for_web = datastore_services.StringProperty(
         required=True, indexed=True)
+    # The list of skill_id that will be used from the topic in the
+    # diagnostic test. In a diagnostic test, any learner can give a test and get
+    # some set of topics as recommendations.
```

Comment:
```suggestion
    # The list of skill IDs that will be used from the topic in the
    # diagnostic test. In a diagnostic test, any learner can give a test and get
    # some set of topics as recommendations.
```

Also, this is a bit hard to read for me, can you try to rephrase?

Reply:
Done